course_info/static/course_info/js/controllers/EnrollmentsController.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+    var app = angular.module('CourseInfo');
+    app.controller('EnrollmentsController', EnrollmentsController);
+
+    function EnrollmentsController($scope, $routeParams, courseInstances, $compile, djangoUrl) {
+        $scope.course_instance_id = $routeParams.course_instance_id;
+        var ci = courseInstances.instances[$scope.course_instance_id];
+        if (angular.isDefined(ci)) {
```

Comment:
Isn't it better to check for ci.title here rather than isDefined(which  determines if the reference to ci is defined)?